coffee_core/src/coffee.rs

```diff
@@ -330,6 +330,19 @@ impl PluginManager for CoffeeManager {
         );
         Err(err)
     }
+
+    async fn search(&mut self, plugin: &str) -> Result<Value, CoffeeError> {
+        for repo in &self.repos {
+            if let Some(plugin) = repo.get_plugin_by_name(plugin) {
+                return Ok(json!({
+                    "repository": repo.url().url_string,
+                    "plugin": serde_json::to_value(&plugin).unwrap(),
```

Comment:
```suggestion
                    "plugin": serde_json::to_value(&plugin)?,
```

Most of the time call unwrap is wrong because will panic the program

Reply:
Alright.
I had to map the error to CoffeeError